src/main/kotlin/eu/europa/ec/eudi/pidissuer/PidIssuerApplication.kt

```diff
@@ -273,10 +273,13 @@ fun beans(clock: Clock) = beans {
             .build()
 
         GetPidDataFromAuthServer(
-            env.getRequiredProperty("issuer.pid.issuingCountry").let(::IsoCountry),
-            clock,
-            keycloak,
-            keycloakProperties.userRealm,
+            issuerCountry = env.getRequiredProperty("issuer.pid.issuingCountry").let(::IsoCountry),
+            issuingJurisdiction = env.getProperty("issuer.pid.issuingJurisdiction"),
+            clock = clock,
+            keycloak = keycloak,
+            userRealm = keycloakProperties.userRealm,
+            generateDocumentNumber = { DocumentNumber(UUID.randomUUID().toString()) },
```

Comment:
Please move the last two generators into the `GetPidDataFromAuthServer`.
No need to be externally provided